src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationModalMap.tsx

```diff
@@ -20,28 +20,25 @@ import {
   MapControlButton,
 } from '../mapButtons/MapButtonContainer'
 
-import {
-  IMAGE_CLASSIFICATION_COLORS,
-  IMAGE_CLASSIFICATION_COLORS as COLORS,
-} from '../../../../library/constants/constants'
+import { IMAGE_CLASSIFICATION_COLORS as COLORS } from '../../../../library/constants/constants'
```

Comment:
Non-blocking: Why make a generic alias for a more specific name. To be less verbose? 

I typically encourage exactness over shortened symbol names.

Reply:
It looks to be to blend into existing code. I get that. If had the bandwidth I'd improve the code with the more descriptive constant name